setup.cfg

```
@@ -127,7 +127,8 @@ all =
     apache-airflow-providers-http>=4.9.0
     apache-airflow-providers-snowflake>=5.3.0
     apache-airflow-providers-sftp
-    apache-airflow-providers-microsoft-azure>=8.5.1
+    # TODO: Increment microsoft-azure version as per the upcoming release for deprecating WasbPrefixSensorAsync
+    apache-airflow-providers-microsoft-azure>=9.0.0rc1
```

```suggestion
    apache-airflow-providers-microsoft-azure>=9.0.0
```
Apply once RC is released